extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -513,30 +513,32 @@ private void setProjectJdk(Map<IPath, IClasspathEntry> classpathMap, List<BuildT
             javaProject.setOption(JavaCore.COMPILER_COMPLIANCE, targetCompatibility);
         }
 
-        String highestJavaVersion = getHighestCompatibleJavaVersion(jvmBuildTarget.getGradleVersion());
-        try {
-            IVMInstall vm = EclipseVmUtil.findOrRegisterStandardVM(
-                targetCompatibility, // expectedVersion
-                sourceCompatibility, // lowestVersion
-                highestJavaVersion,
-                new File(new URI(jvmBuildTarget.getJavaHome())) // fallback jdk
-            );
+        if (!StringUtils.isEmpty(jvmBuildTarget.getJavaHome())) {
```

Comment:
Both gradle version and java version should not empty?

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/974ebc4e114459b65b61c7e8c95634986511ae9f